auth/jwt_test.go

```diff
@@ -13,6 +14,7 @@ import (
 
 type JWTAuthenticatorTestSuite struct {
 	goyave.TestSuite
+	user *TestUser
```

Comment:
@System-Glitch could you help me with this issues? what is the linter complaining about exactly?

Reply:
This is a new linter I enabled yesterday. This linter ensure all structs have minimal padding so they are taking less space. (Learn more [here](https://dave.cheney.net/2015/10/09/padding-is-hard)

Just move that field above `goyave.TestSuite` and that warning will go away.